web/modules/custom/lit_open_platform/src/Plugin/EntityReferenceSelection/LitNodeSelection.php

```diff
@@ -155,7 +155,7 @@ private function processEntities(array $entities, array $books): array {
       }
     }
 
-    if (empty($options['book'])) {
+    if (empty($options['book'] || is_null($options['book']))) {
```

Comment:
I think that `empty(NULL) == false` base on https://www.php.net/manual/en/language.types.boolean.php#language.types.boolean.casting so that will make the last part not reachable.

So should this be
`is_null($options['book'])) || empty($options['book']`

Not 110% sure, but think that would be safe'er